flow-defs/theme.js.flow

```diff
@@ -79,7 +79,7 @@ declare type LinkComponent = React.ComponentType<{
 declare export var AnchorLink: LinkComponent;
 export type ColorScheme = "dark" | "light" | "auto";
 export type ThemeConfig = {
-  skin: Skin,
+  skin: $ReadOnly<Skin>,
```

Comment:
Ahá! Nice catch (thanks for pointing to https://flow.org/en/docs/faq/#toc-why-can-t-i-pass-a-string-to-a-function-that-takes-a-string-number)